client/src/components/ReviewsBreakdown/ReviewsBreakdown.test.jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import ReviewsBreakdown from './ReviewsBreakdown.jsx';
```

Comment:
Super thorough test on for this component. I like how you took the time to create the fake props and tested them in the same way your component would be rendering your actual data. Because both of the mock props are so similar maybe create it outside of the describe block and then add in the parts you are changing inside of the test. For example, your first set of fakeProps would be the initial, and then it looks like you are reusing almost the same one and change a few properties each time so just update those object properties accordingly inside the test.
`    
const fakeProps = {
  overallRatingGroup: {
    ratingGroup: 'Very Positive',
    type: 'positive',
  },
  recentRatingGroup: {
    ratingGroup: null,
    type: null,
  },
  totalRecentReviewCount: 0,
  totalReviewCount: 149,
  enoughRecentReviews: false,
  enoughTotalReviews: true,
};
describe('ReviewsBreakdown component', () => {
  it('Should correctly render review breakdown when the recent review count is 0', () => {
    
    const component = shallow(<ReviewsBreakdown reviewStats={fakeProps} />);

    expect(component.find('.total_rating_group').text()).toEqual('Very Positive');
    expect(component.find('.recent_rating_group').text()).toEqual('0 user reviews');
    expect(component.find('.total_review_count').text()).toEqual(' (149 reviews) ');
    expect(component.find('.recent_review_count').text()).toEqual(' (0 reviews) ');
  });

  it('Should correctly render review breakdown when the recent review count is greater than 0', () => {
    
    fakeProps.recentRatingGroup.ratingGroup = 'Mostly Positive'; //(etc...)

    const component = shallow(<ReviewsBreakdown reviewStats={fakeProps} />);

    expect(component.find('.total_rating_group').text()).toEqual('Very Positive');
    expect(component.find('.recent_rating_group').text()).toEqual('Mostly Positive');
    expect(component.find('.total_review_count').text()).toEqual(' (149 reviews) ');
    expect(component.find('.recent_review_count').text()).toEqual(' (12 reviews) ');
  });
});
`

Definitely not necessary but just a way to prevent code reuse.

Reply:
Super solid suggestion